rwth_upper_body_skeleton_random_walk/launch/fixed.launch

```diff
@@ -0,0 +1,20 @@
+<launch>
+
+<include file ="$(find ground_plane_estimation)/launch/ground_plane_fixed.launch"/>
+<include file ="$(find upper_body_detector)/launch/upper_body_detector.launch"/>
```

Comment:
Again, before merging, this should be considered. @urafi Please provide a minimal launch file in this package that doesn't launch things outside of it. Put "system" launch files (files that launch multiple packages) in https://github.com/strands-project/strands_perception_people/tree/indigo-devel/perception_people_launch, please.